tools/apiview/emitters/cadl-apiview/src/emitter.ts

```diff
@@ -1,6 +1,7 @@
 // See: https://cadlwebsite.z1.web.core.windows.net/docs/extending-cadl/emitters-basics
```

Comment:
Do you have no change to do to handle scalars?

Reply:
I will now 😄  thanks for the reminder!

Reply:
Added